### ISSUE TITLE
During training, the validation performance is always 0.

### ISSUE THREAD

**Issue by taotaoorange** [Microsoft/human-pose-estimation.pytorch#17]
I've been trying to train on coco. The training loss seems normal. However, the validation performance is always 0. Any idea what is happening?

**Comment by taotaoorange** [comment]
It seems it is related to CUDNN #8. It worked after disabling CUDNN. I am using 1080ti. It is a bit weird because in my other projects I do not have this problem with cudnn and batchnorm.  @leoxiaobin

**Comment by ahuirecome** [comment]
Well, it's rather weird. I'v encountered with the same problem. Thank you for your tip!
